### PR TITLE
waves: move API headers to correct place.

### PR DESCRIPTION
**Pull request by lgirdwood**
Move the waves headers to the correct directory.

Signed-off-by: Liam Girdwood <liam.r.girdwood@linux.intel.com>

### CODE REVIEW

**Comment by lgirdwood** [comment]
@stolx can you give this a try with your binaries and let us know.

**Review by dbaluta — APPROVED** [review]

File: src/audio/codec_adapter/CMakeLists.txt [hunk 1]
```diff
@@ -13,6 +13,4 @@ endif()
 if(CONFIG_WAVES_CODEC)
 add_local_sources(sof codec/waves.c)
 sof_add_static_library(MaxxChrome ${CMAKE_CURRENT_LIST_DIR}/lib/release/libMaxxChrome.a)
-# folder with Waves API must be among include directories
-target_include_directories(sof PUBLIC ${CMAKE_CURRENT_LIST_DIR}/../../include/sof/audio/codec_adapter/codec/)
```
Comment (stolx) [inline src/audio/codec_adapter/CMakeLists.txt:17]: SOF with Waves codec will not compile with those changes
as a comment says Waves API folder should be on includes path
new API location is not on includes path
so do not delete this but rather change it to 
`target_include_directories(sof PUBLIC ${CMAKE_CURRENT_LIST_DIR}/../../include/sof/audio/)`
and keep a comment

**Review by stolx — CHANGES_REQUESTED** [review]
please add folder with MaxxEffect API to path, it wont compile without it 
`target_include_directories(sof PUBLIC ${CMAKE_CURRENT_LIST_DIR}/../../include/sof/audio/)`

**Comment by lgirdwood** [comment]
@stolx update pushed with your changes applied..

**Review by stolx — APPROVED** [review]

**Review by dbaluta — APPROVED** [review]

**Review by cujomalainey — APPROVED** [review]

**Comment by cujomalainey** [comment]
Doxygen is known issue, all other CIs are non-informational since this code is not part of CI

**Pull request merged**